pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -25,6 +25,14 @@ buildPythonPackage rec {
     sha256 = "1hqan9h55x9gfdakw554vic68w9gpvhblchwxlw265zxp56hxjrw";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
Explain the raison d'être (pardon my français) or this patch.

Reply:
Added comment.